help/fr/glossary/index.shtml

```diff
@@ -19,8 +19,7 @@
 
 <!-- Sortable tables -->
 <!-- <script src="../../../web/js/sorttable.js"></script> for local testing -->
-<script src="/web/js/sorttable.js"></script>
-
+<script src="/web/js/sorttable.js" type="text/javascript"></scri
```

Comment:
The file `help/fr/glossary/index.shtml` has a bad HTML tag at line 22.

`</scri` should be `</script>`